plugins/applications/src/lib.rs

```diff
@@ -98,12 +123,20 @@ pub fn init(config_dir: RString) -> State {
         }
     };
 
+    // only load execution stats, if needed
+    let execution_stats = if config.use_usage_statistics {
```

Comment:
if we don't keep `use_usage_statistics`, we need to update this to check if `usage_score_multiplier > 0`.

Reply:
See https://github.com/anyrun-org/anyrun/pull/176#discussion_r1687048099